go-apps/meep-rnis/server/rnis_test.go

```diff
@@ -0,0 +1,2093 @@
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+	_, err = sendRequest(http.MethodDelete, "/subscriptions/rab_mod", nil, nil, nil, http.StatusNotImplemented, RabModSubscriptionsSubscrIdDELETE)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+
+	//subscriptions rab rel
+	_, err = sendRequest(http.MethodGet, "/subscriptions/rab_rel", nil, nil, nil, http.StatusNotImplemented, SubscriptionLinkListSubscriptionsRrGET)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+	_, err = sendRequest(http.MethodGet, "/subscriptions/rab_rel", nil, nil, nil, http.StatusNotImplemented, RabRelSubscriptionSubscriptionsGET)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+	_, err = sendRequest(http.MethodPost, "/subscriptions/rab_rel", nil, nil, nil, http.StatusNotImplemented, RabRelSubscriptionSubscriptionsPOST)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+	_, err = sendRequest(http.MethodPut, "/subscriptions/rab_rel", nil, nil, nil, http.StatusNotImplemented, RabRelSubscriptionSubscriptionsPUT)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+	_, err = sendRequest(http.MethodDelete, "/subscriptions/rab_rel", nil, nil, nil, http.StatusNotImplemented, RabRelSubscriptionsSubscrIdDELETE)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+
+	//subscriptions ca reconf
+	_, err = sendRequest(http.MethodGet, "/subscriptions/ca_reconf", nil, nil, nil, http.StatusNotImplemented, SubscriptionLinkListSubscriptionsCrGET)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+	_, err = sendRequest(http.MethodGet, "/subscriptions/ca_reconf", nil, nil, nil, http.StatusNotImplemented, CaReConfSubscriptionSubscriptionsGET)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+	_, err = sendRequest(http.MethodPost, "/subscriptions/ca_reconf", nil, nil, nil, http.StatusNotImplemented, CaReConfSubscriptionSubscriptionsPOST)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+	_, err = sendRequest(http.MethodPut, "/subscriptions/ca_reconf", nil, nil, nil, http.StatusNotImplemented, CaReConfSubscriptionSubscriptionsPUT)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+	_, err = sendRequest(http.MethodDelete, "/subscriptions/ca_reconf", nil, nil, nil, http.StatusNotImplemented, CaReConfSubscriptionsSubscrIdDELETE)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+
+}
+
+func TestSuccessSubscriptionCellChange(t *testing.T) {
+	fmt.Println("--- ", t.Name())
+	log.MeepTextLogInit(t.Name())
+
+	fmt.Println("TestSuccessSubscriptionCellChange()")
+	mod.DbAddress = redisTestAddr
+	err := Init(redisTestAddr, influxTestAddr, true)
+	if err != nil {
+		t.Error("Error initializing test basic procedure")
+	}
+
+	fmt.Println("Set a scenario")
+	updateScenario(testScenario)
```

Comment:
Using a meep-model instance in the UT code will be more flexible to update the active scenario in these tests. For example you could emulate multiple UE mobility or net char events without having multiple hardcoded scenarios. This will also remove UT code from the microservice files.

Reply:
done